toot-relay.go

```diff
@@ -234,7 +261,9 @@ func handler(writer http.ResponseWriter, request *http.Request) {
 		notification.Priority = apns2.PriorityHigh
 	}
 
-	messageChan <- &Message{isProduction, notification, requestLog}
+	unsubscribeUrl := request.Header.Get("Unsubscribe-Url")
+
+	messageChan <- &Message{isProduction, notification, unsubscribeUrl, requestLog}
```

Comment:
If there's no `Unsubscribe-url` header you' send "" along in `unsubscribeUrl`, that OK?

Reply:
The `if` statement at line 81 ensures that the behavior is unchanged in this case. The new `unsubscribed` field in the entry does not cause issues elsewhere if not read.